src/pages/Login.tsx

```diff
@@ -0,0 +1,225 @@
+        <ThemeProvider theme={theme}>
+            <Grid container component="main" sx={{ height: '100vh' }}>
+                <CssBaseline />
+
+                <Grid
+                    item
+                    xs={false}
+                    sm={4}
+                    md={7}
+                    sx={{
+                        backgroundImage: `url(${login_image})`, backgroundRepeat: 'no-repeat',
+                        backgroundColor: (t) =>
+                            t.palette.mode === 'light' ? t.palette.grey[50] : t.palette.grey[900],
+                        backgroundSize: 'cover',
+                        backgroundPosition: 'center',
+                        position: 'relative'
+                    }}>
+
+                    <Box component="div" sx={{
+                        background: theme.palette.primary.light,
+                        position: 'absolute',
+                        top: 0,
+                        bottom: 0,
+                        left: 0,
+                        right: 0
+                    }}>
+                    </Box>
+
+                </Grid>
+
+                <Grid
+                    item xs={12}
+                    sm={8}
+                    md={5}
+                    component={Paper}
+                    elevation={6}
+                    square
+                    sx={{
+                        display: "flex",
+                        justifyContent: "center",
+                        alignItems: "center",
+                    }}>
+
+                    <Box
+                        sx={{
+                            my: 8,
+                            mx: 4,
+                            display: 'flex',
+                            flexDirection: 'column',
+                            alignItems: 'center',
+                        }}
+                    >
+
+                        <Box component="div" sx={{ width: "228px", height: "84px" }}>
+                            <Box
+                                component="img"
+                                src={intercars_logo}
+                                alt="intercars-logo"
+                                sx={{
+                                    display: 'block',
+                                    width: '100%',
+                                    height: 'auto',
+                                    marginBottom: '1.5em'
+                                }}
+                            />
+                        </Box>
+
+                        <Box component="div" sx={{
+                            fontSize: '1.5em',
+                            fontWeight: 800,
+                            padding: '0 2em',
+                            textAlign: 'center',
+                            margin: '2em 0',
+                        }}>
+                            Вътрешна система за менажиране на доставки
+                        </Box>
+
+
+                        <Typography component="h1" variant="h5" sx={{
```

Comment:
Same as above, and remove `component` prop and `fontSize` styles and just use variant. You have different variants to choose from so 1 will always get close to what exactly you need.